### DISCUSSION TITLE
HTTPS not checking

### DISCUSSION THREAD

**Discussion by GunvirDhesi** [community/community#42226] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

After talking with the bot, it says everything checks out and nothing is wrong with my A records. However, the box is still not ticking.

**Comment by queenofcorgis** [comment]
Hi @GunvirDhesi , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
